core/src/test/java/com/klaytn/caver/scenario/RoleBasedAccountUpdateAccountIT.java

```diff
@@ -0,0 +1,117 @@
+package com.klaytn.caver.scenario;
+
+import com.klaytn.caver.crypto.KlayCredentials;
+import com.klaytn.caver.methods.response.KlayTransactionReceipt;
+import com.klaytn.caver.tx.model.AccountUpdateTransaction;
+import com.klaytn.caver.tx.model.TransactionTransformer;
+import com.klaytn.caver.tx.type.TxTypeFeeDelegate;
+import org.junit.Test;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.Random;
+
+import static org.junit.Assert.assertEquals;
+
+public class RoleBasedAccountUpdateAccountIT extends RoleBasedAccountScenario {
+
+    private final ReceiptChecker getAccountUpdateReceiptChecker = (TransactionTransformer transactionTransformer, KlayTransactionReceipt.TransactionReceipt transactionReceipt) -> {
+        assertEquals("0x1", transactionReceipt.getStatus());
```

Comment:
If you check update account using getAccountKey API(https://docs.klaytn.com/bapp/json-rpc/api-references/klay/account#klay_getaccountkey), your test will be more accurate

Reply:
Wow thanks i'll reflect your review